actionpack/lib/action_controller/metal/strong_parameters.rb

```diff
@@ -998,7 +998,7 @@ def initialize_copy(source)
   # It provides an interface for protecting attributes from end-user
   # assignment. This makes Action Controller parameters forbidden
   # to be used in Active Model mass assignment until they have been
-  # whitelisted.
+  # permitted.
```

Comment:
"forbidden until they have been permitted" is in tautology territory. I think this particular one meant "passed through a whitelist", not "added to a whitelist", so "filtered" might fit better?